packages/dom/src/dom/is-rtl.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import getComputedStyle from './get-computed-style';
+
+/**
+ * Whether the element's text direction is right-to-left.
+ *
+ * @param {Element} element The element to check.
+ *
+ * @return {boolean} True if rtl, false if ltr.
+ */
+export default function isRTL( element ) {
```

Comment:
is the existing wp.i18n.isRTL not enough?

Reply:
This works in any environment and it's contextual to the element. Different elements may have different directions. Generally, I think it's good to avoid globals.

Reply:
We used to have different ways of checking RTL and we consolidate on one recently and I fear this puts us a bit back to more inconsistency.

Now, if the per element logic is something we really need in this use case, I think it's good to have, otherwise I feel we should just consolidate using `wp.i18n.isRTL`.

Reply:
Then perhaps we should revert to not exposing the function but leaving the check by element as it was before.